bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/editor/targetComponent/TelescopePosTableWidget.java

```diff
@@ -1137,16 +1137,17 @@ void setSelectedRow(final Point location) {
      * Selects the relevant row in the table.
      */
     private void _setSelectedRow(final int index) {
-        if ((index < 0) || (index >= getRowCount())) return;
-        getSelectionModel().setSelectionInterval(index, index);
+        if ((index < 0) || (index >= getRowCount())) {
+            getSelectionModel().setSelectionInterval(0, 0);
+        } else {
+            getSelectionModel().setSelectionInterval(index, index);
+        }
```

Comment:
I take this to mean that when you first select any target node it should begin with the base position table row selected.